oxidation/client/src/locales/fr-FR.json

```
@@ -165,6 +193,13 @@
             "lastLoginMinutes": "il y a une minute | il y a {minutes} minutes",
             "lastLoginHours": "il y a une heure | il y a {hours} heures",
             "lastLoginDays": "il y a un jour | il y a {days} jours"
+        },
+        "filesize": {
+            "bytes": "o",
+            "kilobytes": "Ko",
+            "megabytes": "Mo",
+            "gigabytes": "Go",
+            "terabytes": "To"
```

Should we use the SI notation ?

<https://fr.wikipedia.org/wiki/Pr%C3%A9fixe_binaire>

```suggestion
            "bytes": "octet",
            "kilobytes": "KiB",
            "megabytes": "MiB",
            "gigabytes": "GiB",
            "terabytes": "TiB"
```

Or use a kept the `o` suffix

<https://fr.wikipedia.org/wiki/Octet>

```suggestion
            "bytes": "octet",
            "kilobytes": "Kio",
            "megabytes": "Mio",
            "gigabytes": "Gio",
            "terabytes": "Tio"
```

---

See previous comment, taken from https://github.com/Scille/parsec-cloud/blob/master/parsec/core/gui/file_size.py